extension/chrome/elements/compose.ts

```diff
@@ -171,8 +171,8 @@ export class ComposeView extends View {
       await this.renderModule.fetchReplyMeta(Object.keys(storage.sendAs!));
     }
     if (this.isReplyBox) { // reply
-      if (this.threadId && !this.ignoreDraft && storage.drafts_reply && storage.drafts_reply[this.threadId]) {
-        this.draftId = storage.drafts_reply[this.threadId]; // there may be a draft we want to load
+      if (this.threadId && !this.draftId && !this.ignoreDraft && storage.drafts_reply && storage.drafts_reply[this.threadId]) {
+        this.draftId = storage.drafts_reply[this.threadId]; // there may be a legacy draft we want to load
```

Comment:
Interesting - this means we should make another issue to delete this line, and potentially others, maybe 6 months later.

Reply:
Done: https://github.com/FlowCrypt/flowcrypt-browser/issues/3329